content/galacticsilkroad/journey.md

```diff
@@ -0,0 +1,1030 @@
+
+His Journey had officially begun.
+
+## Bio-age 23 years, 11 months
+
+Taryn was above manual labor. He was smart, he was cultured. He had a
+plan for his Journey. He'd saved up plenty of nakas and had been living
+off them for the past three months. *Three bio-months*, he reminded
+himself. Most of the time had been spent in a nearby hub, Nexus Relay.
+It was located near the midpoint between his home planet of Vectra and
+its closest neighboring inhabited planet, Novara.
+
+He was about 15 lightyears from home. At relativistic speeds, it had
+only taken a total of 4 bio-weeks in total, split up into a few stops
+for entertainment at smaller Relays along the way. But Nexus was the
+real deal. He'd been docked for about four days now and was starting to
+adapt to the station's day/night cycle.
+
+His parents had sent him monthly letters, which meant that he now had
+about 180 messages from them. They understood the time dilation well.
+They should, of course. It had allowed them to live 1,000 chrono-years
+already. They knew that, for him, it had only been a few months. They
+never demanded any responses. Even stranger, their letters seemed to
+have a timeless quality. They would mention some day-to-day events, but
+they knew these would be unimportant to him. By the time he read them,
+whatever silly dramas existed back on Vectra would be long resolved.
+
+But to be honest, he'd stopped reading the letters a few weeks back,
+just skimming them. He had more important things to do.
+
+"Another round for my friends!" he called out. Cheers from around the
+bar greeted him. He pulled out his comm and casually dropped another 5
+million nakas on the drinks. Nexus Relay wasn't as cheap as Vectra,
+which had an AME--Average Monthly Expenditure--of 30 million nakas.
+Nexus was a Relay Hub, which meant lots of visitors, lots of imports,
+and lots of incoming money. Here, AME was closer to 100 million nakas.
+
+He tried to avoid it, but he glimpsed the remaining balance on his comm.
+It was down to 100 million nakas, give or take. He'd started this
+journey with a billion. *No matter,* he thought. *Once I get used to
+life on the station, I'll start trading, and I'll be golden.* Vectra was
+an agricultural planet, and had some of the most delicious dried fruit
+in the colonized galaxy. He'd sold half his inventory on the Vectra
+Relay Station #3 for 200 million nakas, and farther out from Vectra it
+should be even more valuable.
+
+"It's certainly none of my business to tell a customer how to spend his
+money," said the older man behind the bar. "But you have the look of
+someone fresh on his Journey. You may want to hold onto your nakas, you
+never know when you'll need them."
+
+"Thanks, but I think I'll be OK," said Taryn, downing his drink. With
+that, he stood up from the bar, walked over to the table with the higher
+female-to-male ratio in the bar, and asked if anyone wanted a refill. He
+wasn't planning on sleeping alone tonight.
+
```

Comment:
I think we need some kind of break indicator here. Like a visible line or three centered asterisks or something like that. We hare having a change of scene within a single bio-age section, and it's confusing without an indicator.

Reply:
Looks like they all got removed when copying Markdown from Google Drive. I've added them back.